Union with EmptySet

```julia
julia> ∪(Singleton([1.]), ∅)
ERROR: AssertionError: sets in a union must have the same dimension
```
Possible solution: `@neutral(UnionSet, EmptySet)`

--- a/src/UnionSet.jl
+++ b/src/UnionSet.jl
@@ -33,6 +33,9 @@ end
 # convenience constructor without type parameter
 UnionSet(X::S1, Y::S2) where {N<:Real, S1<:LazySet{N}, S2<:LazySet{N}} = UnionSet{N, S1, S2}(X, Y)
 
+# EmptySet is the neutral element for UnionSet
+@neutral(UnionSet, EmptySet)
+
 # Universe is the absorbing element for UnionSet
 @absorbing(UnionSet, Universe)
 
@@ -237,6 +240,9 @@ Type that represents the set union of a finite number of convex sets.
 struct UnionSetArray{N<:Real, S<:LazySet{N}}
     array::Vector{S}
 end
+
+# EmptySet is the neutral element for UnionSetArray
+@neutral(UnionSetArray, EmptySet)
 
 # Universe is the absorbing element for UnionSetArray
 @absorbing(UnionSetArray, Universe)

--- a/test/unit_UnionSet.jl
+++ b/test/unit_UnionSet.jl
@@ -15,6 +15,14 @@ for N in [Float64, Rational{Int}, Float32]
     U2 = swap(UXY)
     @test UXY.X == U2.Y && UXY.Y == U2.X
 
+    # neutral and absorbing elements
+    E = EmptySet{N}()
+    @test neutral(UnionSet) == neutral(UnionSetArray) == EmptySet
+    @test UnionSet(B1, E) == UnionSet(E, B1) == B1
+    U = Universe{N}(2)
+    @test absorbing(UnionSet) == absorbing(UnionSetArray) == Universe
+    @test UnionSet(B1, U) == UnionSet(U, B1) == U
+
     for U in [UXY, Uarr]
         # dimension
         @test dim(U) == dim(B1)

